presto-parser/src/main/java/com/facebook/presto/sql/parser/SqlParser.java

```diff
@@ -233,6 +236,7 @@ public void exitNonReserved(SqlBaseParser.NonReservedContext context)
             context.getParent().removeLastChild();
 
             Token token = (Token) context.getChild(0).getPayload();
+            warningConsumer.accept(new ParsingWarning("Non-reserved word used", token.getLine(), token.getCharPositionInLine()));
```

Comment:
Haven't checked if the following would work but possible will:
```java
format("Non-reserved word used: %s", token.getText())

Reply:
It does, thanks!